redbot/cogs/reports/reports.py

```diff
@@ -314,13 +315,27 @@ async def on_raw_reaction_add(self, payload):
 
     @commands.Cog.listener()
     async def on_message(self, message: discord.Message):
+
+        to_remove = []
+
         for k, v in self.tunnel_store.items():
-            topic = _("Re: ticket# {1} in {0.name}").format(*k)
+
+            guild, ticket_number = k
+            if await self.bot.cog_disabled_in_guild(self, guild):
+                to_remove.append(k)
+                continue
+
+            topic = _("Re: ticket# {ticket_number} in {guild.name}").format(
+                ticket_number=ticket_number, guild=guild
+            )
             # Tunnels won't forward unintended messages, this is safe
             msgs = await v["tun"].communicate(message=message, topic=topic)
             if msgs:
                 self.tunnel_store[k]["msgs"] = msgs
 
+        for key in to_remove:
+            self.tunnel_store.pop(key, None)
```

Comment:
react close won't work here, had to add a method that doesn't assume that we're closing because an event was related to that particular tunnel object.

Reply:
You're right, I missed the fact that we need to send message to both sides of the tunnel. Looks good now.